x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/action/TransportStopTransformAction.java

```diff
@@ -398,7 +398,9 @@ private void waitForTransformStopped(
                     if (stillRunningTasks.size() > 0) {
                         message.append("Could not stop the transforms ");
                         message.append(stillRunningTasks);
-                        message.append(" as they timed out.");
+                        message.append(" as they timed out[");
```

Comment:
```suggestion
                        message.append(" as they timed out [");
```

Perhaps a space between `out` and `[`